homeassistant/components/decora_wifi/translations/en.json

```diff
@@ -0,0 +1,22 @@
+{
```

Comment:
Delete this file from the PR, will be created once merged and translations are done.

Reply:
Done.